src/components/link/Link.jsx

```diff
@@ -99,14 +99,15 @@ export default class Link extends React.Component {
         fill: this.props.fontColor,
         fontSize: this.props.fontSize,
         fontWeight: this.props.fontWeight,
+        textAnchor: "middle",
```

Comment:
For retro-compatibility to move forward with this PR I'll prefer that we allow clients to pass this prop alongside [`node.color` and others](https://danielcaldas.github.io/react-d3-graph/docs/#node-color). Please refer to the [docs](https://danielcaldas.github.io/react-d3-graph/docs/#node-color) and [a sample past PR](https://github.com/danielcaldas/react-d3-graph/pull/303) to know what files require an update to make this work

Reply:
Just to clarify, shouldn't the prop be passed alongside other link props, such as [`link.fontColor`](https://danielcaldas.github.io/react-d3-graph/docs/#link-font-color)? Similarly, should clients be able to pass the `startOffset` (which is also currently locked to 50%)? That change seems to be of a similar nature to allowing clients to pass `textAnchor` as a prop, since it was previously locked to `"middle"`. Glad to combine any or all of these changes as you see fit.

Reply:
Sorry for the confusion. Yes it should be `link.textAnchor`, but the changeset should be similar. [Here's a PR](https://github.com/danielcaldas/react-d3-graph/pull/120) that introduced a property in the `link` for reference